packages/beacon-node/src/api/rest/base.ts

```diff
@@ -97,7 +97,12 @@ export class RestApiServer {
       if (err instanceof ErrorAborted || err instanceof NodeIsSyncing) return;
 
       const {operationId} = res.context.config as RouteConfig;
-      this.logger.error(`Req ${req.id} ${operationId} error`, {}, err);
+
+      if (err instanceof ApiError) {
+        this.logger.warn(`Req ${req.id} ${operationId} failed`, {reason: err.message});
```

Comment:
@wemeetagain I want to establish this pattern that for non-critical issues we use the terms "failed"  and "reason" instead of `error` as based on my observations the term "error" scares a lot of users, especially less technical ones. What do you think?

Reply:
:+1: :+1:

Reply:
i think the stack trace is very useful, but we can move the log to `debug` will full trace, an additional `warn` log in this format is acceptable

Reply:
> but we can move the log to debug will full trace, an additional warn log in this format is acceptable

that's also what I was considering, sounds like a reasonable compromise. Not sure if it makes sense here though in case of `ApiError`, if someone feels strongly about having a stack trace for those we should probably log it as well to `debug`